script.js

```diff
@@ -32,18 +40,28 @@ function addActiveNavButtonClass(index) {
 	document.getElementById(index).classList.add('carousel__indicator--active')
 }
 
+// EVENT LISTENERS
+
 ui.buttonPrevious.addEventListener('click', () => {
```

Comment:
As we are already separating code for creating carousel into a different file it would probably be best to also separate the navigation handling as well. 🤔